src/adapters/Authentication/index.ts

```diff
@@ -0,0 +1,19 @@
+import baseAdapter from 'adapters/Base';
+import { Config } from 'config';
+
+const AuthenticationAdapter = () => {
+  const signIn = (email: string, password: string) =>
+    baseAdapter.post('oauth/token', {
+      clientId: Config.clientId,
+      clientSecret: Config.clientSecret,
+      grantType: 'password',
+      email: email,
+      password: password,
+    });
+
+  return { signIn };
```

Comment:
why don't we return directly the `signIn`?

Reply:
@nvminhtue I will get the error if returning the `signIn` directly 💭 
![image](https://github.com/manh-t/react-survey/assets/60863885/e39b0bb2-7b8e-47f0-8074-ac0b80fcd531)